litho-testing/src/main/java/com/facebook/litho/testing/specmodels/MockSpecModel.java

```diff
@@ -202,6 +203,11 @@ private MockSpecModel(
     mEnclosedSpecModel = enclosedSpecModel;
   }
 
+  @Override
+  public TypeElement getOriginatingElement() {
+    return null;
```

Comment:
Can this be null under normal (production) circumstances? If not, is there a way to create a no-op `TypeElement` instead to make sure we keep this non-nullable throughout?

Reply:
@passy Normally it should never be `null`! I can take a look to check if I can create a no-op `TypeElement`! I believe that this would be a good solution since there is no `TestingProcessor` registered for incremental annotation processing, thus avoiding any possible issues in tests.

Reply:
@passy Added a `NoOpTypeElement` and removed `@Nullable` annotation from `SpecModel`. Tests were green locally. Waiting for CI as well.